applications/annotate_points.md

```diff
@@ -0,0 +1,314 @@
+# annotating videos with napari
+
+In this tutorial, we will use napari (requires version 0.3.2 or greater) to make a simple GUI application for annotating points in videos. This GUI could be useful for making annotations required to train algorithms for markless tracking of animals (e.g., [DeepLabCut](http://www.mousemotorlab.org/deeplabcut)). In this tutorial, we will cover creating and interacting with a Points layer with properties (i.e., labels for the points), connecting custom UI elements to events, and creating custom keybindings.
+
+At the end of this tutorial, we will have created a GUI for annotating points in videos that we can simply call by:
+
+```python
+im_path = '<path to directory with data>/*.png'
+output = '<path to directory with data>/annotations.csv'
+point_annotator(im_path, labels=['ear_l', 'ear_r', 'tail'], output_path=output)
+
+```
+that looks like this:
+
+![image]({{ '/assets/tutorials/point_annotator_demo.gif' | relative_url }})
```

Comment:
fwiw, it would be nice if everyone got on board with video tags instead of gifs. They use up less space (so download faster), *and* look smoother. An example from my blog post:

```html
<video width="90%" autoplay loop muted playsinline>
  <source src="/napari/napari-blobs-720p.mp4" type="video/mp4" />
  <source src="/napari/napari-blobs-720p.ogg" type="video/ogg" />
  <img src="/napari/napari-blobs-720p-frame.jpg"
      title="Your browser does not support the video tag." />
</video>
```

As a reminder, markdown is a superset of html, so this works fine within a markdown document.

Reply:
for video tags does that mean we make `.mp4` instead? say using a tool like the quicktime screen recorder instead of `gifs`? My impression was they were always much larger, but I'm open to suggestions/ ideas here

Reply:
The video files also end up being larger for me. Perhaps we can circle back to this in the future.